production_configs/20210416_v0.7.3_prod5_trans_80_local_taicut_8_4/log_onsite_mc_r0_to_dl3_20210416_v0.7.3_prod5_trans_80_local_taicut_8_4.yml

```diff
@@ -0,0 +1,2438 @@
+r0_to_dl1:
```

Comment:
This are the full logs for a single MC prod. I don't think this should go inside the repo. 
Note that there has been some renaming of these filenames, but they should all contain the prod_id within their name.

Reply:
I must have taken the wrong files then. Will DM you.

Reply:
Ok I think this is fixed